apps/patient/src/views/Pharmacy.tsx

```diff
@@ -55,42 +55,62 @@ const query = (method, data) =>
     })
   })
 
-function getHours(hoursData) {
-  const now = dayjs()
+export const getHours = (
```

Comment:
I feel this could live in utils, would be great isolate for unit tests this in the future.

Reply:
ah true

Reply:
yeah, tried some unit tests, but nx was giving me a headache, might need your help on setting those up